src/icon.c

```diff
@@ -115,8 +117,6 @@ GdkPixbuf *get_pixbuf_from_icon(const char *iconname)
                         start = end + 1;
                 } while (*(end) != '\0');
         }
-        if (!pixbuf)
-                LOG_W("Could not load icon: '%s'", iconname);
```

Comment:
This error message is still important. If there is no icon file named as `iconname` in your `settings.icon_path`, then still no icon would appear, but also no message would get logged.

I would refine the message and use `LOG_W("No icon found for: '%s'", iconname);`. Then it would make sense together with the message from `get_pixbuf_from_file`.